ci/infra/testrunner/platforms/platform.py

```diff
@@ -131,6 +144,12 @@ def _create_node_log_dir(ip_address, node_type, log_dir_path):
 
         return node_log_dir_path
 
+    def _add_node(self, node_type):
+        """Expected to be overridden"""
```

Comment:
Maybe let's raise some exception there? In case if someone uses that method by accident when it's still not implemented.

Reply:
Yeah I'll take care of that.